index.html

```diff
@@ -395,30 +409,34 @@ <h3 class="subtitle is-5">Don't hesitate, ask anything</h3>
             <fieldset class="field">
               <label class="label" for="username">Username</label>
               <div class="control has-icons-left has-icons-right">
-                <input id="username" class="input is-success" type="text" name="username">
-                <span class="icon is-small is-left">
+                <input id="username" class="input is-success" type="text" name="username"
+                  placeholder="Enter your username" aria-describedby="usernameHelp"
+                  aria-invalid="false">
+                <span class="icon is-small is-left" aria-hidden="true">
                   <i class="fas fa-user"></i>
                 </span>
-                <span class="icon is-small is-right">
+                <span class="icon is-small is-right" aria-hidden="true">
                   <i class="fas fa-check"></i>
                 </span>
               </div>
-              <p class="help is-success">This username is available</p>
+              <p id="usernameHelp" role="alert" class="help is-success">This username is
+                available</p>
             </fieldset>
 
             <fieldset class="field">
               <label class="label" for="email">Email</label>
               <div class="control has-icons-left has-icons-right">
                 <input id="email" class="input is-danger" type="email"
-                  placeholder="Enter your email" name="email" value="hello@">
-                <span class="icon is-small is-left">
+                  placeholder="Enter your email" name="email" value="hello@"
+                  aria-describedby="emailHelp" aria-invalid="true">
+                <span class="icon is-small is-left" aria-hidden="true">
                   <i class="fas fa-envelope"></i>
                 </span>
-                <span class="icon is-small is-right">
+                <span class="icon is-small is-right" aria-hidden="true">
                   <i class="fas fa-exclamation-triangle"></i>
                 </span>
               </div>
-              <p class="help is-danger">This email is invalid</p>
+              <p id="emailHelp" role="alert" class="help is-danger">This email is invalid</p>
```

Comment:
Also this is a good case to use aria-describedby attr
Check example here: https://www.tpgi.com/using-aria-describedby-to-provide-helpful-form-hints/

Reply:
But it was added for input:
<img width="469" alt="Screenshot 2024-02-16 at 20 50 04" src="https://github.com/yurassic/accessibility-training/assets/10263616/1781cd84-ac26-4c39-9963-474c2f10d9ae">

Reply:
oops, sorry, I missed that. I was checking the HW for about 2 hours, so I think I was just tired in the end

Reply:
NP 😄